.gitignore

```diff
@@ -1,7 +1,7 @@
 # Node stuff
 npm-debug.log
 node_modules
-package-lock.json # Favour yarn.lock, heroku fails if both included
+package-lock.json
```

Comment:
I think the comment is still worth keeping, can we just move it to the line above?

Reply:
yes, that will work. I just thought it's not necessary. I'll add it back.